packages/ui/src/components/Button/index.tsx

```diff
@@ -20,17 +22,21 @@ export function Button({
   className = '',
   children,
   onClick,
-  size,
+  size = '',
+  full = false,
+  icon,
 }: ButtonProps) {
+  const block = full ? 'block' : '';
```

Comment:
here you want to try this https://styled-components.com/docs/basics#adapting-based-on-props